src/components/MappingEvents/Statistics.js

```diff
@@ -1,18 +1,20 @@
+// @flow
+
 import React from 'react';
 import { t } from 'ttag';
 import styled from 'styled-components';
 
-import MapPinWithPlusIcon from './MapPinWithPlusIcon';
-import BellIcon from './BellIcon';
+import MapPinWithPlusIcon from '../icons/ui-elements/MapPinWithPlusIcon';
+import BellIcon from '../icons/ui-elements/BellIcon';
 import { UserIcon } from '../icons/ui-elements/index';
 
-interface Props {
-  mappedPlacesCount: number;
-  invitedParticipantCount: number;
-  startDate: Date;
-  endDate: ?Date;
-  className: string;
-}
+type Props = {
+  mappedPlacesCount: number,
+  invitedParticipantCount: number,
+  startDate: Date,
```

Comment:
-1: startDate is optional in the mapping event planner

Reply:
Done